src/schedule-and-details/instructors-section/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuid } from 'uuid';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Button } from '@edx/paragon';
+import { Add as AddIcon } from '@edx/paragon/icons';
+import InstructorContainer from './instructor-container';
+
+import ScheduleSubHeader from '../schedule-sub-header';
+import messages from './messages';
+
+const InstructorsSection = ({ instructors, onChange }) => {
+  const intl = useIntl();
+  const newInstructor = {
+    bio: '',
+    image: '',
+    name: '',
+    organization: '',
+    title: '',
+  };
+
+  const handleChange = (value, idx, field) => {
```

Comment:
`updateInstructorField` maybe better

Reply:
it is the simplest component with instructor logic, I think `handleChange` would be enough.